src/processors/babel.ts

```diff
@@ -26,3 +26,8 @@ export default (options?: Options.Babel): PreprocessorGroup => ({
     };
   },
 });
+
+// both for backwards compat with old svelte-preprocess versions
+// (was only default export once, now is named export because of transpilation causing node not to detect the named exports of 'svelte-preprocess' otherwise)
+export default babel;
```

Comment:
Do we still need the default export here? IIRC these were re-exported with names in the package entrypoint.

Reply:
I find that a tad unlikely, but that's an unfortunately good point 🥲